conf/application.conf

```diff
@@ -24,7 +24,7 @@ play {
   }
   filters.headers {
     # Unset some of the security filters enabled in datastore.Filters
-    contentSecurityPolicy = null
+    contentSecurityPolicy = "default-src 'self'; script-src 'self' 'unsafe-eval' www.googletagmanager.com 'sha256-NA81o2FV8jh0UizSPsBz2qEGpXfap54Eghuo7vxrH8g='; style-src 'self' 'sha256-DOu86drLfwUr1Wcsx/wxfqAogK7tFvJGjVmF/300H/M=' 'sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU='; connect-src 'self' ws://localhost:9002 app.ovly.co; frame-src webknossos.olvy.co; img-src 'self' data:"
```

Comment:
It would be helpful to document which hash belongs to which script and how to recompute the hash if the script will ever be changed.